test/e2e/config/e2e_conf.yaml

```diff
@@ -95,7 +137,7 @@ intervals:
   default/wait-delete-cluster: ["20m", "10s"]
   default/wait-machine-upgrade: ["50m", "10s"]
   default/wait-machine-remediation: ["30m", "10s"]
-  default/wait-vm-state: ["20m", "100ms"]
+  default/wait-vm-state: ["10m", "100ms"]
```

Comment:
Is this necessary if yes then why ? In case it  is this just a leftover from debugging please revert the change?